### DISCUSSION TITLE
add sorting by date uploaded

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#19752] [Ideas] [answered]
the option will sort by when beatmaps were first uploaded to osu

**Comment by TheFantasticLoki** [comment]
To further expand upon this idea maybe a sort by date originally downloaded cause you can generally remember how long ago you first downloaded a map but you almost never remember when you update them. The current sor by date added is always changing the order of my maps cause some maps will need to be updated then they go to the very start of the list instead of staying where you remember. So rather than sorting by date added it feels more like date modified.

**Reply by peppy** [reply]
Downloaded date is "date added". It is NOT affected by updates.

**Comment by peppy** [comment, marked as answer]
Already in the works, requires some further work to be added and of use (see  #9468 https://github.com/ppy/osu/pull/19233 / https://github.com/ppy/osu-onlinedb-generator/pull/5
